addon/components/column-resize-handle.js

```diff
@@ -0,0 +1,79 @@
+import Ember from 'ember';
+import $ from 'jquery';
```

Comment:
Not sure really. I can change it to `const { $ } = Ember` if you like. I just figured since the jquery shim is included in ember-cli-shims I should use it when I use the global jquery object